src/applications/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -16,20 +16,18 @@ import {
 
 export function getEnrollmentData() {
   return dispatch =>
-    apiRequest(
-      '/post911_gi_bill_status',
-      null,
-      response => {
+    apiRequest('/post911_gi_bill_status')
+      .then(response => {
         recordEvent({ event: 'post911-status-success' });
         return dispatch({
           type: GET_ENROLLMENT_DATA_SUCCESS,
           data: response.data.attributes,
         });
-      },
-      response => {
+      })
+      .catch(response => {
         recordEvent({ event: 'post911-status-failure' });
         const error =
-          response.errors.length > 0 ? response.errors[0] : undefined;
+          response?.errors?.length > 0 ? response.errors[0] : undefined;
```

Comment:
The optional chaining is so that the tests which are mocking `fetch` so that it calls `Promise.reject` without an object with an `errors` property will still pass.  I didn't want to get too deep into the weeds on refactoring the tests.